services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/Utils.kt

```diff
@@ -76,3 +86,43 @@ fun BusinessId.isValidBusinessId(): Boolean {
 }
 
 fun List<CustomerWithContacts>.ordererCount() = flatMap { it.contacts }.count { it.orderer }
+
+fun userContact(name: String?, email: String?): HankeUserContact? {
```

Comment:
This could use an expression body like the other one.

Reply:
Fixed.